packages/test-harness/src/launchVscodeAndRunTests.ts

```diff
@@ -35,7 +35,7 @@ export async function launchVscodeAndRunTests(extensionTestsPath: string) {
     // NB: We include the exact version here instead of in `test.yml` so that
     // we don't have to update the branch protection rules every time we bump
     // the legacy VSCode version.
-    const vscodeVersion = useLegacyVscode ? "1.75.1" : "stable";
+    const vscodeVersion = useLegacyVscode ? "1.79.2" : "stable";
```

Comment:
This is the version that introduced `jsonl` language id. Because this PR introduces our first jsonl tests, it fails on older VSCode versions. Given this version was released a year ago, we can make this our legacy version per our policy